images/oc-build-deploy-dind/openshift-templates/cli-persistent/deployment.yml

```
@@ -103,6 +103,8 @@ objects:
               secret:
                 defaultMode: 420
                 secretName: lagoon-sshkey
+            - name: ${PERSISTENT_STORAGE_NAME}-twig
+              type: emptyDir
```

@twardnw this is probably wrong, should be
```suggestion
              emptyDir: {}
```